last-rev-llc/lastrev-libraries
# Filtered collection pagination

#### Description

Filtered Collection Pagination 
- Used swr to have easier data fetching logic and take advantage of all benefits of it (: 


---

##### 🔗 Preview URL
[Deploy Preview URL](https://deploy-preview-XXX.netlify.app/)

##### 🔬 How to test
* Go to blogs landing page
* Use various filter to see blogs list 
* Verify all blogs have images

##### 📸  Screenshots _(if applicable)_
Load more
![image](https://user-images.githubusercontent.com/9321946/131616010-aae1b781-9c67-4540-982f-7e8b4388700f.png)
![image](https://user-images.githubusercontent.com/9321946/131616023-89bcb874-6d93-4e7a-906c-acec52e53b95.png)

Load more no longer showing when there's no more pages
![image](https://user-images.githubusercontent.com/9321946/131616046-42d989d9-8a49-40e4-8c5a-1429e258e24c.png)

---
 
##### Changes include
